packages/workflow/jest.config.js

```diff
@@ -21,6 +21,8 @@ function create(settings) {
   const userInclude = settings.configuration.development.babelInclude;
   const includes = ['@av', ...userInclude].join('|');
 
+  const userJestOverrides = settings.configuration.development.jestOverrides
```

Comment:
We should add this new key to [the schema](https://github.com/Availity/availity-workflow/blob/f31b3dbcc8d41a98b056f55c097a38e33d0b636f/packages/workflow/settings/schema.js#L4) and include a description in [the `README`](https://github.com/Availity/availity-workflow/blob/master/README.md)

Since we're returning the merged object this isn't as big a deal, but `lodash.merge` will apply these changes to the source object, in this case `config`. I think `return _.merge({}, config, userJestOverrides);` would be better here, that way we aren't changing the `config` object, what do you think?